package.json

```diff
@@ -26,48 +26,48 @@
     "arrowParens": "avoid"
   },
   "dependencies": {
-    "@babel/core": "^7.18.6",
-    "@babel/plugin-proposal-decorators": "^7.18.6",
-    "@babel/plugin-proposal-export-default-from": "^7.18.6",
+    "@babel/core": "^7.19.3",
+    "@babel/plugin-proposal-decorators": "^7.19.3",
+    "@babel/plugin-proposal-export-default-from": "^7.18.10",
     "@babel/plugin-transform-react-inline-elements": "^7.18.6",
-    "@babel/preset-env": "^7.18.6",
+    "@babel/preset-env": "^7.19.4",
     "@babel/preset-react": "^7.18.6",
-    "@enact/dev-utils": "^5.0.1",
-    "@storybook/addon-actions": "^6.5.10",
-    "@storybook/addon-controls": "^6.5.10",
-    "@storybook/addon-docs": "^6.5.10",
-    "@storybook/addon-toolbars": "^6.5.10",
+    "@enact/dev-utils": "^5.0.2",
+    "@storybook/addon-actions": "^6.5.12",
+    "@storybook/addon-controls": "^6.5.12",
+    "@storybook/addon-docs": "^6.5.12",
+    "@storybook/addon-toolbars": "^6.5.12",
     "babel-plugin-dev-expression": "^0.2.3",
     "css-loader": "^6.7.1",
     "less": "^4.1.3",
-    "less-loader": "^8.1.1",
+    "less-loader": "^11.1.0",
     "mini-css-extract-plugin": "^2.6.1",
-    "node-polyfill-webpack-plugin": "^1.1.4",
-    "postcss": "^8.4.14",
+    "node-polyfill-webpack-plugin": "^2.0.1",
+    "postcss": "^8.4.18",
     "postcss-flexbugs-fixes": "^5.0.2",
     "postcss-global-import": "^1.0.6",
-    "postcss-loader": "^6.2.1",
+    "postcss-loader": "^7.0.1",
```

Comment:
same here

Reply:
Me neither. Let's not update this module at this time.
Please revert this change.

Reply:
Me neither. `postcss-loader@7.0.1`'s minimum supported Node.js version is 14.15.0.
Please revert this and let's investigate this update in an additional ticket.

Reply:
ok